src/containers/SortingView.tsx

```diff
@@ -47,16 +47,15 @@ type Props = StateProps & DispatchProps & OwnProps & RouteComponentProps
 
 type CalcStatus = 'waiting' | 'computing' | 'finished'
 
-type SelectedUnitIds = {[key: string]: boolean}
-
 const SortingView: React.FunctionComponent<Props> = (props) => {
   const { sortingViews, sortingUnitViews, documentInfo, sorting, sortingId, recording, curationDispatch, onSetSortingInfo, onSetExternalUnitMetrics, extensionsConfig, sortingUnitMetrics, hither } = props
   const { documentId, feedUri, readOnly } = documentInfo;
   const [sortingInfoStatus, setSortingInfoStatus] = useState<CalcStatus>('waiting');
   const [externalUnitMetricsStatus, setExternalUnitMetricsStatus] = useState<CalcStatus>('waiting');
   // const [selection, dispatchSelection] = useReducer(updateSelections, {focusedUnitId: null, selectedUnitIds: {}});
-  const [selectedUnitIds, setSelectedUnitIds] = useState<SelectedUnitIds>({})
-  const [focusedUnitId, setFocusedUnitId] = useState<number | null>(null)
+  const [selection, selectionDispatch] = useReducer(sortingSelectionReducer, defaultSortingSelection)
```

Comment:
Here is where we maintain the selection state for the sorting. This never gets to the global redux state, and I think that is okay for now.